src/party_menu_list_items.c

```diff
@@ -0,0 +1,830 @@
+    PartyMenu_DisableMainScreenBlend_AfterYesNo();
+    if (partyMenu->monsDrawState[partyMenu->partyMonIndex].capsule == 0) {
+        partyMenu->args->selectedAction = PARTY_MENU_ACTION_RETURN_0;
+        *pState = PARTY_MENU_STATE_BEGIN_EXIT;
+    } else {
+        PartyMenu_PrintMessageOnWindow34(partyMenu, msg_0300_00179, TRUE);
+        partyMenu->yesCallback = PartyMenu_ConfirmRemoveCapsuleYes;
+        partyMenu->noCallback = PartyMenu_ConfirmRemoveCapsuleNo;
+        partyMenu->afterTextPrinterState = PARTY_MENU_STATE_YES_NO_INIT;
+        *pState = PARTY_MENU_STATE_WAIT_TEXT_PRINTER;
+    }
+}
+
+int PartyMenu_ConfirmRemoveCapsuleYes(PartyMenuStruct *partyMenu) {
+    Pokemon_RemoveCapsule(Party_GetMonByIndex(partyMenu->args->party, partyMenu->partyMonIndex));
+    partyMenu->args->selectedAction = PARTY_MENU_ACTION_RETURN_0;
+    return PARTY_MENU_STATE_BEGIN_EXIT;
+}
+
+int PartyMenu_ConfirmRemoveCapsuleNo(PartyMenuStruct *partyMenu) {
+    G2_BlendNone();
+    ClearFrameAndWindow2(&partyMenu->windows[PARTY_MENU_WINDOW_ID_34], TRUE);
+    PartyMenu_PrintMessageOnWindow32(partyMenu, msg_0300_00029, TRUE);
+    thunk_Sprite_SetPalIndex(partyMenu->sprites[PARTY_MENU_SPRITE_ID_CURSOR], 0);
+    return PARTY_MENU_STATE_1;
+}
+
+static void PartyMonContextMenuAction_Switch(PartyMenuStruct *partyMenu, int *pState) {
+    partyMenu->secondCursorActive = 1;
+    partyMenu->softboiledDonorSlot = partyMenu->partyMonIndex;
+    thunk_Sprite_SetPalIndex(partyMenu->sprites[PARTY_MENU_SPRITE_ID_CURSOR], 0);
+
+    s16 x, y;
+    Sprite_GetPositionXY(partyMenu->sprites[PARTY_MENU_SPRITE_ID_CURSOR], &x, &y);
+    Sprite_SetPositionXY(partyMenu->sprites[PARTY_MENU_SPRITE_ID_SWITCH_MON_CURSOR], x, y);
+    Set2dSpriteAnimSeqNo(partyMenu->sprites[PARTY_MENU_SPRITE_ID_SWITCH_MON_CURSOR], sub_0207B5EC(partyMenu->args->unk_25, partyMenu->softboiledDonorSlot) + 2);
+    Set2dSpriteVisibleFlag(partyMenu->sprites[PARTY_MENU_SPRITE_ID_SWITCH_MON_CURSOR], TRUE);
+    sub_0207A7F4(partyMenu, partyMenu->softboiledDonorSlot);
+    ClearFrameAndWindow2(&partyMenu->windows[PARTY_MENU_WINDOW_ID_33], TRUE);
+    PartyMenu_DeleteContextMenuAndList(partyMenu);
+    PartyMenu_DisableMainScreenBlend_AfterYesNo();
+    PartyMenu_PrintMessageOnWindow32(partyMenu, msg_0300_00031, TRUE);
+    *pState = PARTY_MENU_STATE_SELECT_SWITCH_MON;
+}
+
+void sub_0207FBC8(PartyMenuStruct *partyMenu) {
+    partyMenu->secondCursorActive = 0;
+    Set2dSpriteVisibleFlag(partyMenu->sprites[PARTY_MENU_SPRITE_ID_SWITCH_MON_CURSOR], FALSE);
+    if (partyMenu->partyMonIndex < PARTY_SIZE) {
+        sub_0207A7F4(partyMenu, partyMenu->partyMonIndex);
+    }
+    PartyMenu_SetTopScreenSelectionPanelVisibility(partyMenu, FALSE);
+    sub_0207A7F4(partyMenu, partyMenu->softboiledDonorSlot);
+    PartyMenu_PrintMessageOnWindow32(partyMenu, msg_0300_00029, TRUE);
+}
+
+void sub_0207FC1C(PartyMenuStruct *partyMenu) {
+    u16 *bg2Tilemap;
+    u16 *bg1Tilemap;
+    s8 spC;
```

Comment:
try to not use sp terminology once the func is decompiled

Reply:
not a big enough deal to fix, as they'll be renamed once the function is better understood